packages/components/src/components/empty-state/empty-state.scss

```diff
@@ -4,6 +4,5 @@
     flex: 1;
     flex-direction: column;
     align-items: center;
-    justify-content: center;
```

Comment:
As discussed with the design team, They don't want the content of the empty states to be center aligned.